browser/templates/browser/investigator.html

```diff
@@ -79,6 +79,23 @@
         </div>
     </div>
     {% endwith %}
+
+    {% with form.institution as field %}
+    <div class="row">
+        <div class="col-xs-12">
+            <div class="form-group">
+                {% for error in field.errors %}
+                <div class="alert alert-warning">{{ error }}</div>
+                {% endfor %}
+                <label>{{ field.label }}</label>
+                {{ form.institution_search|addcss:"form-control input-sm" }}
+                <div class="list-group" id="institution-results-container"></div>
+                <p class="text-muted">{{ field.help_text }}</p>
+            </div>
+        </div>
+    </div>
```

Comment:
I'm very confused by this. Why do you use two fields? You only seem to be setting the institution_search field, what is the purpose of the institution field?

Reply:
The institution field was not being used. So, I have removed it.